src/BlazorWebView/src/WindowsForms/BlazorWebView.cs

```diff
@@ -170,7 +170,9 @@ private void StartWebViewCoreIfPossible()
 			// We assume the host page is always in the root of the content directory, because it's
 			// unclear there's any other use case. We can add more options later if so.
 			string appRootDir;
+#pragma warning disable IL3000 // 'System.Reflection.Assembly.Location.get' always returns an empty string for assemblies embedded in a single-file app. If the path to the app directory is needed, consider calling 'System.AppContext.BaseDirectory'.
 			var entryAssemblyLocation = Assembly.GetEntryAssembly()?.Location;
+#pragma warning restore IL3000
```

Comment:
@simonrozsival maybe we could file an issue about this one?

Reply:
Just to make sure: do you mean changing `Environment.CurrentDirectory` to `System.AppContext.BaseDirectory` or suppressing the warning for getting the assembly location?

Reply:
An issue to change it to `System.AppContext.BaseDirectory`, as it seems more appropriate. You can publish WinForms and WPF as single file, and `Environment.CurrentDirectory` doesn't seem correct.

Reply:
https://github.com/dotnet/maui/issues/21750